attributes/default.rb

```diff
@@ -126,7 +72,7 @@
 default['kibana']['nginx']['server_name'] = 'kibana'
 
 #<> The nginx configuration source
-default['kibana']['nginx']['source'] = node['kibana']['version'] == '4' ? 'nginx4.conf.erb' : 'nginx.conf.erb'
+default['kibana']['nginx']['source'] = node['kibana']['version'] == '4' ? 'nginx4.conf.erb' : 'nginx5.conf.erb'
```

Comment:
Probably need to make this conditional test a regex (`=~ /^4/`) to match all 4.x versions